src/knockout/templates/header.html

```diff
@@ -14,14 +12,11 @@ <h3 data-bind="css: css.title"><!-- ko template: { name: 'survey-string', data:
           <h5 data-bind="css: css.description"><!-- ko template: { name: 'survey-string', data: locDescription } --><!-- /ko --></h5>
       </div>
       <!-- /ko -->
-
+      
       <!-- ko if: isLogoAfter -->
-      <div data-bind="css: logoClassNames">
-        <img data-bind="css: css.logoImage, attr: { src: locLogo.koRenderedHtml, width: logoWidth ? logoWidth : undefined, height: logoHeight ? logoHeight : undefined, alt: locTitle.renderedHtml }, style: { objectFit: logoFit }">
-      </div>
-      <div class="sv-logo--right-tail"></div>
+      <!-- ko component: { name: getElementWrapperComponentNameByName('sv-logo-image'), params: getElementWrapperComponentDataByName('sv-logo-image') } -->
```

Comment:
I thought it should be `getElementWrapperComponentName(survey, 'logo-image')` instead of `getElementWrapperComponentNameByName('sv-logo-image')` . Element - survey - reason - 'logo-image'

Reply:
Yes, this is probably reasonable to replace  `getElementWrapperComponentNameByName('sv-logo-image')` to `getElementWrapperComponentName(survey, 'logo-image')`. When I added  `getElementWrapperComponentNameByName` method the `getElementWrapperComponentName` wasn't exist